evap/evaluation/templates/confirmation_text_modal.html

```diff
@@ -10,14 +10,14 @@ <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
             <form id="{{ modal_id }}Form">
                 <div class="modal-body">
                     {{ question|safe }}
-                    <br /><br />
-                    <div class="mb-3">
+                    <br />
```

Comment:
can you completely remove this linebreak? the margin should already do the work.